docs/dev/api_integration.md

```diff
@@ -1,13 +1,27 @@
 ---
 title: Create an API Integration
 type: documentation
+further_reading:
+- link: "/api/latest/using-the-api/"
+  tag: "Documentation"
+  text: "Learn how to use the Datadog API"
+- link: "/developers/authorization/"
+  tag: "Documentation"
+  text: "Learn about using OAuth for API integrations"
+- link: "/developers/"
+  tag: "Documentation"
+  text: "Learn how to develop on the Datadog platform"
 ---
 
 ## Overview
 
-Use [Datadog API endpoints][1] to enrich a customer's experience by submitting data from your backend and pulling data from a user's Datadog account. API integrations work well for building a connector between Datadog and another SaaS platform. Partners write and host their code within their environment.
+This page walks Technology Partners through how to create a Datadog API integration on the [**Integrations** page][22]. 
```

Comment:
```suggestion
This page walks Technology Partners through how to create a Datadog API integration. 
```
It's rare, but you technically could create an api integration on the markeplace